app/loading.tsx

```diff
@@ -1,12 +1,12 @@
-import style from "./loading.module.scss";
+import style from './loading.module.scss';
 
-const Loading = () => {
+function Loading() {
```

Comment:
Why is this done?
This change seems uneccessary.

Reply:
https://stackoverflow.com/questions/75587906/why-airbnb-style-guide-force-eslint-rule-react-function-component-definition

Reply:
Davko lets chat on the next meeting about implementing a style guide. THis tiket is just for prettier and adding it into husky

Reply:
@MarkoDavkovski As @edichoska said, it's important to be consistent throughout the project, regardless of which approach is picked.

If this change is done by eslint, we should investigate the rule that forces this change and maybe disable it.

Reply:
Absolutely, consistency across the project is crucial, as both @edichoska and the ticket description emphasize. We opted for the Airbnb style guide, which indeed mandates the use of function components over arrow functions for components.
If there are areas where my implementation can be enhanced or if the application of the Airbnb rules through eslint has unintentionally enforced any undesirable changes, I'm open to suggestions.

Reply:
I wasn't aware of the Airbnb style guide decision.
I have nothing against your changes other than the fact that I didn't understand them and wanted to make sure everything is according to plan. 

You're on the right track. Please proceed with this so we can finalize this change.
https://github.com/learnhubmkd/app/pull/61#pullrequestreview-1926549314